v2/esp-entity-table.ts

```diff
@@ -59,7 +59,7 @@ export class EntityTable extends LitElement {
         } as entityConfig;
         this.entities.push(entity);
         this.entities.sort((a, b) => (a.name < b.name ? -1 : 1));
-        this.has_controls ||= this.control(entity).length||0>0
+        this.has_controls ||= this.control(entity).values.length > 0;
```

Comment:
This isn't going to work because when the entity is a switch the `values.length` will be 0. Maybe a combination of both will work. Something like this?

`this.has_controls ||= this.control(entity).length > 0 || this.control(entity).values.length > 0;`

Reply:
I think the problem here is that `control(..)` is returning different types instead of being consistent. Perhaps it should always return a list so we can easily check the length